ansible/script/keystone.sh

```diff
@@ -153,6 +205,10 @@ install(){
         docker pull opensdsio/opensds-authchecker:latest
         docker run -d --privileged=true --net=host --name=opensds-authchecker opensdsio/opensds-authchecker:latest
         docker cp "$TOP_DIR/../../conf/keystone.policy.json" opensds-authchecker:/etc/keystone/policy.json
+        sudo apt-get install -y python-openstackclient
```

Comment:
suggestion: don't not use apt-get go install python-openstackclient, becuase the the client install by apt is not the newset the one, and often cause version issue,please use pip install instead.

Reply:
At first, I thought apt would be better because openstack related libraries are not core components of opensds, and it's better relying on distribution standard packages.

But, after looking into the current installer playbook, I noticed that 'ansible/roles/osdsdock/scenarios/cinder_standalone.yml' also uses pip.
So, I agree using pip is better in this case, and I will update this PR later.